src/pages/public_registration/PublicPublicationContext.tsx

```diff
@@ -499,6 +497,12 @@ const PublicAudioVisualPublicationDialogContent = ({
       <Typography paragraph>{publisher.name}</Typography>
       <Typography variant="h3">{t('registration.resource_type.artistic.catalogue_number')}</Typography>
       <Typography paragraph>{catalogueNumber}</Typography>
+      {isrc.value !== null ? (
+        <>
+          <Typography variant="h3">{t('registration.resource_type.artistic.music_score_isrc')}</Typography>
+          <Typography paragraph>{hyphenateIsrc(isrc.value)}</Typography>
+        </>
+      ) : null}
```

Comment:
Ser ut som vi alltid viser overskrift for andre inputs her. Litt usikker på om det er bevisst, eller fordi det meste er eller har vært required, men det er kanskje naturlig å vise at det ikke har noen verdi også..?
I så fall:
```suggestion
      <Typography variant="h3">{t('registration.resource_type.artistic.music_score_isrc')}</Typography>
      {isrc.value && <Typography paragraph>{hyphenateIsrc(isrc.value)}</Typography>}
```

Reply:
etter en kjapp sjekk ser jeg vi har en god miks av 3 løsninger rundt dette... Vise bare header (tomt innhold), vise `-` for å vise at det ikke er noe der, og skjule hele avsnittet. 😅 Så der burde vi egentlig hatt en standard løsning for alt, men det er en annen sak :)

Reply:
går for det samme syns jeg. Det blir uansett ulikt noen plasser nå 😅 kan evt lage en oppgave for å gjøre dette likt for alle typene, så spørre cecilie om hva vi ønsker